src/main/java/edu/cornell/kfs/rass/batch/service/impl/RassUpdateServiceImpl.java

```diff
@@ -445,11 +455,44 @@ protected UserSession buildSessionForSystemUser() {
         maintenanceDocument.getDocumentHeader().setDocumentDescription(
                 buildDocumentDescription(newBo, maintenanceAction, objectDefinition));
         
+        try {
         maintenanceDocument = (MaintenanceDocument) documentService.routeDocument(
                 maintenanceDocument, annotation, null);
+        } catch(ValidationException ve) {
```

Comment:
A space is needed after the "catch" keyword.

Reply:
formatted